infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -232,7 +232,7 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable,
         uint256 finalPrice = ethToXai(finalEthPrice);
 
         // Confirm the final price does not exceed the expected cost
-        require(_expectedCost <= finalPrice, "Price Exceeds Expected Cost");
```

Comment:
@XDapps How come the testcases did not catch that. Do we need to refactor the tests ?

Reply:
The tests did not catch this because the test is/was > _or_ =. Since the test case was sending the exact expected cost, the test was passing.